magento/pwa-studio
# [Docs] Revise and publish PWA Studio fundamentals intro and project setup

## Description

This PR revises and moves the PWA Studio fundamentals Intro and Project setup topics from @rossmc out of the drafts folder and into the appropriate publish folder.

## Related Issue

Closes PWA-298

## Acceptance 

Any core developers

### Verification Stakeholders

Any core developers

### Verification Steps

1. Navigate to the docs directory
1. Run linting on the new topics:

   ```bash
   npm run lint src/tutorials/pwa-studio-fundamentals/
   ```

1. Verify no linting issues
1. Start the HTML preview server: `npm start`
1. Navigate to the **Tutorials** section
1. Verify **PWA Studio fundamentals** and **Project setup** are listed as links on the left nav
1. Verify links lead to the correct pages

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
